projects/angular/src/forms/datepicker/date-range-start-input.ts

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { AfterViewInit, Directive, EventEmitter, HostBinding, HostListener, Input, Output } from '@angular/core';
+import { of } from 'rxjs';
+import { filter, switchMap } from 'rxjs/operators';
+
+import { ClrDateInputBase } from './date-input';
+import { DayModel } from './model/day.model';
+import { DatepickerFocusService } from './providers/datepicker-focus.service';
+import { datesAreEqual } from './utils/date-utils';
+
+@Directive({
+  selector: '[clrRangeStartDate]',
```

Comment:
```suggestion
  selector: '[clrStartDateInput]',
```

Same as above, I think "range" is redundant here. Also, it should end in "input" like `clrDateInput`. Please apply this to the end date input directive and the validator directives as well. And please rename the classes to match.